site/src/pages/components/invokers.explainer.mdx

```diff
@@ -380,10 +380,14 @@ that this list is ordered and higher rules take precedence:
 | `<details>`     | `'toggle'`            | If the `<details>` is `open`, then close it, otherwise open it                       |
 | `<details>`     | `'open'`              | If the `<details>` is not `open`, then open it                                       |
 | `<details>`     | `'close'`             | If the `<details>` is `open`, then close it                                          |
-| `<select>`      | `'auto'`              | Open the `<select>` option picker UI                                                 |
-| `<select>`      | `'showPicker'`        | Open the `<select>` option picker UI                                                 |
-| `<input>`       | `'auto'`              | Call `.showPicker()` on the invokee                                                  |
-| `<input>`       | `'showPicker'`        | Call `.showPicker()` on the invokee                                                  |
+| `<select>`      | `'auto'`              | If the `<select>` is `open`, then close it, otherwise open it                        |
+| `<select>`      | `'togglePicker'`      | If the `<select>` is `open`, then close it, otherwise open it                        |
+| `<select>`      | `'showPicker'`        | If the `<select>` is not `open`, then open it                                        |
+| `<select>`      | `'closePicker'`       | If the `<select>` is `open`, then close it                                           |
```

Comment:
Should this be closePicker or hidePicker?

It's hidePopover but close dialog and details?

Reply:
Give it's show and not openPicker I'm leaning towards this actually being hidePicker?